bin/commands/plugins.js

```diff
@@ -1,612 +1,101 @@
-        }
-
-        if (options.set) {
-          return setPluginConfiguration(plugin, options.set, dbService, kuzzleConfiguration);
-        }
-
-        if (options.unset) {
-          return unsetPluginConfiguration(plugin, options.unset, dbService, kuzzleConfiguration);
-        }
-
-        if (options.replace) {
-          return replacePluginConfiguration(plugin, options.replace, dbService, kuzzleConfiguration);
         }
-
-        if (options.remove) {
-          return removePlugin(plugin, dbService, kuzzleConfiguration);
-        }
-
-        if (options.activate) {
-          return dbService
-            .update(kuzzleConfiguration.pluginsManager.dataCollection, plugin, {activated: true})
-            .then(() => getPluginConfiguration(plugin, dbService, kuzzleConfiguration));
-        }
-
-        if (options.deactivate) {
-          return dbService
-            .update(kuzzleConfiguration.pluginsManager.dataCollection, plugin, {activated: false})
-            .then(() => getPluginConfiguration(plugin, dbService, kuzzleConfiguration));
-        }
-      })
-      .then(() => {
-        release();
-
-        if (options.install) {
-          if (plugin) {
-            console.log(clcOk('███ kuzzle-plugins: Plugin ' + plugin + ' installed'));
-          }
-          else {
-            console.log(clcOk('███ kuzzle-plugins: Plugins installed'));
-          }
-        }
-
-        process.exit(0);
-      })
-      .catch(error => {
-        release();
-
-        /*
-         If elasticsearch returns with a "Service temporary unavailable" error,
-         we retry until it's ready
-         */
-        if (error.status === 503) {
-          console.log('The database seems to not be ready yet. Retrying in 5s...');
-          return setTimeout(() => pluginsManager(plugin, options), 5000);
-        }
-
-        console.error(clcError('Error: '), error);
-        process.exit(error.status);
-      });
-  });
-};
-
-/**
- * Creates the internalIndex in the Kuzzle database, if it
- * doesn't already exists
- *
- * @param db
- * @param kuzzleConfiguration
- */
-function initializeInternalIndex(db, kuzzleConfiguration) {
-  return db
-    .createInternalIndex()
-    .then(() => {
-      /*
-        Disables mapping on the 'config' object to avoid conflicts
-        between different plugins
-       */
-      return db.updateMapping(
-        kuzzleConfiguration.pluginsManager.dataCollection,
-        {
-          properties: {
-            config: {
-              enabled: false
-            }
-          }
-        }
-      );
+      } else {
+        console.log(res);
+      }
+      process.exit(0);
     })
     .catch(err => {
-      // ignoring error if it's raised because the index already exists
-      if (err.status === 400) {
-        return q();
-      }
-
-      return q.reject(err);
+      console.error(clcError(err.message));
+      process.exit(1);
```

Comment:
hmm I don't understand :/
Actually, we are in bin/commands/plugins.js here, and you do the exit ;)
What I ask, is, why you do `process.exit(1)` instead of `process.exit(err.status)` ?